app.rb

```diff
@@ -0,0 +1,89 @@
+require './book'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There is no books in the list...'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'There is no people in the list...'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age #{person.age}" }
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    choice = gets.chomp
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    case choice
+    when '1'
+      print 'Has parent permission? [Y/N]: '
```

Comment:
- [ ] Kindly make sure that a user can successfully create a student or teacher, I see this error on the console 👍🏿 

<img width="665" alt="person" src="https://github.com/lucy-sees/Libra/assets/69638013/cc897c37-b009-4049-bb94-cd3d8feb66e3">